src/Components/DnDLayout/GridTile.tsx

```diff
@@ -31,6 +31,7 @@ import { getWidget } from '../Widgets/widgetDefaults';
 import { useAtomValue } from 'jotai';
 import classNames from 'classnames';
 import HeaderIcon from '../Icons/HeaderIcon';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
```

Comment:
Last nit, I don't see this import used anywhere now.

Reply:
I see it being used here https://github.com/RedHatInsights/widget-layout/pull/91/files#diff-cada88f315638722c5aec42832c3e0435ebb8c61b87fafdc68e6de78b2035b20R59

Reply:
Sorry my bad!